pysindy/optimizers/trapping_sr3.py

```diff
@@ -237,6 +236,21 @@ def __init__(
         )
         if thresholder.lower() not in ("l1", "l2", "weighted_l1", "weighted_l2"):
             raise ValueError("Regularizer must be (weighted) L1 or L2")
+        if eta is None:
+            print(
```

Comment:
This should be a warning instead of a print statement.

Reply:
looks like you fixed this already, thanks!